doc/content/gateways/troubleshooting/_index.md

```diff
@@ -140,3 +140,39 @@ Try solving this issue by:
 
 - Checking your internet connection
 - Using {{% tts %}} cluster closest to the location of your gateway
+
+## What is Radio Configuration?
+
+There are three types of radio configuration. All types are typically defined in the same `station.conf` file.
+
+**Hardware Specific**: configuration unique to the hardware. This is defined in a file shipped with the gateway and should not be modified. Examples are `clksrc` and RSSI offset.
+
+**LoRaWAN Regional**: configuration according to LoRaWAN Regional Parameters. This is common to all gateways using the same Regional Parameters, and is provided by the network server automatically when using {{% lbs %}}. It must be configured manually for UDP gateways. Examples are data rates and mandatory channels.
+
+**User Defined**: configuration which the user is free to choose. Examples are non-mandatory channels and sub-bands.
+
+For an example configuration file, see [Semtech's {{% lbs %}} documentation](https://doc.sm.tc/station/gw_v1.5.html#single-board-sample-configuration).
+
+## What modes of authentication are supported?
+
+{{% lbs %}} defines bi-directional authentication at server side ({{% tts %}} and client side (gateway). Server side is authenticated with a trusted certificate and client side is authenticated either by HTTP token or certificate. See [Semtech's documentation](https://doc.sm.tc/station/authmodes.html#authentication-modes).
```

Comment:
`({{% tts %}})` missing a closing parenthesis. 

I would also omit the use of "side" from "server side" and "client side".